src/main/notifications/index.ts

```diff
@@ -70,11 +70,16 @@ class NotificationManager {
             log.debug('notification click', serverName, mention.uId);
 
             this.allActiveNotifications.delete(mention.uId);
-            MainWindow.show();
-            if (serverName) {
+
+            // Show the window after navigation has finished to avoid the focus handler
+            // being called before the current channel has updated
+            const focus = () => {
```

Comment:
Not sure if relevant but before we were checking on `serverName` before trying to move to that view, do we want to keep that check?

Reply:
No it was just a null-check that didn't have any relevance, so I removed it.